src/coreclr/jit/hwintrinsicxarch.cpp

```diff
@@ -2361,7 +2361,7 @@ GenTree* Compiler::impSpecialIntrinsic(NamedIntrinsic        intrinsic,
             }
             else if (simdSize == 64)
             {
-                if (varTypeIsByte(simdBaseType))
+                if (varTypeIsByte(simdBaseType) && (!compExactlyDependsOn(InstructionSet_AVX512VBMI)))
```

Comment:
nit: Unnecessary parentheses:
```suggestion
                if (varTypeIsByte(simdBaseType) && !compExactlyDependsOn(InstructionSet_AVX512VBMI))
```

Reply:
Part of different PR